frontend/src/pages/follows/FollowsPageController.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import { useParams } from "react-router-dom"
+import FollowsPageView from "./FollowsPageView"
+import followers from './mock-followers'
+import follows from './mock-follows'
+
+/**
+ * This page renders a user's followers. It also contains
+ * a header and the navigation footer.
+ */
+const FollowsPageController = () => {
+    // gets the id from the current url 
+    const { id } = useParams()
```

Comment:
This will be undefined, because the route used is /user/:username/follows, so it should be:

```js
const { username } = useParams()
```

You can also remove the parseInt below

Reply:
Ok I changed this